src/main/steps/sibling/placement-order/get.test.ts

```diff
@@ -0,0 +1,187 @@
+import { mockRequest } from '../../../../test/unit/utils/mockRequest';
+import { mockResponse } from '../../../../test/unit/utils/mockResponse';
+import { generateContent } from '../placement-order-number/content';
+
+import PlacementOrderGetController from './get';
+
+describe('PlacementOrderGetController', () => {
+  let controller;
+  let req;
+  let res;
+
+  beforeEach(() => {
+    Date.now = jest.fn(() => +new Date('2021-01-01'));
+    controller = new PlacementOrderGetController(__dirname + '../../common/template', generateContent);
+    req = mockRequest({ session: { userCase: { placementOrders: [] } } });
```

Comment:
this file should be in accordance with sibling palcementOrders and not userCase.placementOrders